storm/src/main/storm/mesos/util/MesosCommon.java

```diff
@@ -105,7 +105,7 @@ public static String taskId(String nodeid, int port) {
   }
 
   public static String supervisorId(String nodeid, String topologyId) {
-    return String.format("%s-%s", nodeid, topologyId);
+    return String.format("%s%s%s", nodeid, DEFAULT_MESOS_COMPONENT_NAME_DELIMITER, topologyId);
```

Comment:
@erikdw If the mesos component name delimiter is configurable then we should probably make this use `MesosCommon.getMesosComponentNameDelimiter()`. My question is, what is the advantage of making it configurable in the first place? This gives the user ability to possibly break storm-mesos.